Dropdown/Dropdown.js

```diff
@@ -184,37 +196,77 @@ const DropdownBase = kind({
 		const opened = !disabled && open;
+		const dropdownButton = (skin === 'silicon');
 
 		return (
 			<div {...rest}>
-				<Item
-					{...rest}
-					css={css}
-					disabled={hasChildren ? disabled : true}
-					onClick={opened ? onClose : onOpen}
-				>
-					<Icon slot="slotAfter" className={css.icon} size="small">{open ? 'arrowlargeup' : 'arrowlargedown'}</Icon>
-					{title}
-				</Item>
-				<Transition
-					className={transitionContainerClassname}
-					visible={opened}
-					direction={transitionDirection}
-				>
-					<ContainerDiv className={dropdownListClassname} spotlightDisabled={!open} spotlightRestrict="self-only">
-						<Scroller className={css.scroller}>
-							<Group
-								className={css.group}
-								childComponent={Item}
-								itemProps={{size: 'small'}}
-								onSelect={onSelect}
-								selected={selected}
-							>{children || []}</Group>
-						</Scroller>
-					</ContainerDiv>
-				</Transition>
+				{dropdownButton &&
+				<div className={dropdownButtonClassname}>
+					<Button
+						{...rest}
+						css={css}
+						disabled={hasChildren ? disabled : true}
+						onClick={opened ? onClose : onOpen}
+					>
+						{title}
+						<Icon slot="slotAfter" className={css.icon} size="small">{open ? 'arrowlargeup' : 'arrowlargedown'}</Icon>
+					</Button>
+					<Transition
+						className={transitionContainerClassname}
+						visible={opened}
+						direction={transitionDirection}
+					>
+						<ContainerDiv className={dropdownListClassname} spotlightDisabled={!open} spotlightRestrict="self-only">
+							<Scroller className={css.scroller}>
+								<Group
+									className={css.group}
+									childComponent={RadioItem}
+									itemProps={{size: 'small', className: css.dropDownListItem, css: css}}
+									onSelect={onSelect}
+									selected={selected}
+									selectedProp="selected"
+								>
+									{children || []}
+								</Group>
+							</Scroller>
+						</ContainerDiv>
+					</Transition>
+				</div>
+				}
+				{!dropdownButton &&
+					<React.Fragment>
+						<Item
+							{...rest}
+							css={css}
+							disabled={hasChildren ? disabled : true}
+							onClick={opened ? onClose : onOpen}
+						>
+							<Icon slot="slotAfter" className={css.icon} size="small">{open ? 'arrowlargeup' : 'arrowlargedown'}</Icon>
+							{title}
+						</Item>
+						<Transition
+							className={transitionContainerClassname}
+							visible={opened}
+							direction={transitionDirection}
+						>
+							<ContainerDiv className={dropdownListClassname} spotlightDisabled={!open} spotlightRestrict="self-only">
+								<Scroller className={css.scroller}>
+									<Group
+										className={css.group}
+										childComponent={Item}
+										itemProps={{size: 'small'}}
+										onSelect={onSelect}
+										selected={selected}
+									>
+										{children || []}
+									</Group>
+								</Scroller>
+							</ContainerDiv>
+						</Transition>
+					</React.Fragment>
+				}
```

Comment:
I don't like the two pair of the components depending on the `silicon` skin.
Please check my suggestion and let me know your thoughts.

The commit based on your PR: https://github.com/enactjs/agate/commit/cf3829c4f9207055c71ea072d23bf087f8fea909

```suggestion
const [DropDownButton, wrapperProps, groupProps] = (skin === 'silicon') ? [
	Button,
	{className: dropdownButtonClassname, component: 'div'},
	{childComponent: RadioItem, className: css.dropDownListItem, css, itemProps: {size: 'small', selectedProp: 'selected'}}
] : [
	Item,
	{component: React.Fragment},
	{childComponent: Item, itemProps: {size: 'small'}}
];

return (
	<div {...rest}>
		<ComponentOverride {...wrapperProps}>
			<DropDownButton
				{...rest}
				css={css}
				disabled={hasChildren ? disabled : true}
				onClick={opened ? onClose : onOpen}
			>
				<Icon slot="slotAfter" className={css.icon} size="small">{open ? 'arrowlargeup' : 'arrowlargedown'}</Icon>
				{title}
			</DropDownButton>
			<Transition
				className={transitionContainerClassname}
				visible={opened}
				direction={transitionDirection}
			>
				<ContainerDiv className={dropdownListClassname} spotlightDisabled={!open} spotlightRestrict="self-only">
					<Scroller className={css.scroller}>
						<Group
							className={css.group}
							onSelect={onSelect}
							selected={selected}
							{...groupProps}
						>
							{children || []}
						</Group>
					</Scroller>
				</ContainerDiv>
			</Transition>
		</ComponentOverride>
	</div>
);
```

Reply:
applied the suggested change with minor adjustments. see  story ticket